src/cmds/create_plots.py

```diff
@@ -93,28 +106,27 @@ def main():
     else:
         sk_seed = token_bytes(32)
         log(
-            f"Using sk_seed {sk_seed.hex()}. Note that sk seed is now generated randomly, as opposed "
-            f"to from keys.yaml. If you want to use a specific seed, use the -s argument."
+            f"Using sk_seed {sk_seed.hex()}. Note that sk seed is now generated randomly. "
```

Comment:
Any chance we can refactor the actual plotting logic to some other location so that things like `tests/block_tools.py`can use them too?

Reply:
Yeah I should do that.